views/content-cross-linked.phtml

```diff
@@ -1,3 +1,16 @@
 <?= $data->content ?>
 <hr>
-<p><em>Also published on <a href="<?= $data->medium_post->url ?>">Medium</a>.</em></p>
+<p>
+	<em>
+		<?php
+		printf(
```

Comment:
@majelbstoat Thanks for the styling feedback. It's cool if you want to enforce your own conventions in the plugin, it just happens to be that the [official WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) are the opposite of some of these you're suggesting, which is how myself and others are used to styling things :smiley: